data/nm/legislature/Joseph-Sanchez-5d176173-a628-499c-8181-c61fc27231b8.yml

```diff
@@ -0,0 +1,20 @@
+id: ocd-person/5d176173-a628-499c-8181-c61fc27231b8
+name: Joseph Sanchez
+given_name: Joseph
+family_name: Sanchez
+email: Joseph.Sanchez@nmlegis.gov
+image: https://www.nmlegis.gov/Images/Legislators/House/HSAJO.jpg
+party:
+- name: Democratic
```

Comment:
This person should already have a file, we just need to add the new role:
- type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:nm/government
  district: '40'
They previously served in House District 40 from 01/01/2019-12/31/2020